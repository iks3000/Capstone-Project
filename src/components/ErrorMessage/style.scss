@import "../../assets/styles/_variables.scss";
@import "../../assets/styles/_extends.scss";

.error-message {
    @extend %content-text;
    width: 100%;
    height: 20vh;
    display: flex;
    align-items: center;
    justify-content: center;
    color: $red;
}