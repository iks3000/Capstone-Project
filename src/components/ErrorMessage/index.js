import './style.scss';

const ErrorMessage = () => {
    return (
        <div className="error-message">
            Something went wrong; please review your server connection!
        </div>
    )
}

export default ErrorMessage;