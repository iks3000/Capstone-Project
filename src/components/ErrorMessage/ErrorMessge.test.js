import React from "react";
import { shallow } from "enzyme";
import ErrorMessage from "./index";

it("should render ErrorMessage component", () => {
    expect(shallow(<ErrorMessage />)).toMatchSnapshot();
});
