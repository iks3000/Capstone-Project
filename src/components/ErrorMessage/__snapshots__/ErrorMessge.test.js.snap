// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ErrorMessage component 1`] = `
<div
  className="error-message"
>
  Something went wrong; please review your server connection!
</div>
`;
